SPECS/sm.spec

```diff
@@ -388,9 +418,9 @@ Manager and some other packages
 
 
 %changelog
-* Fri Nov 29 2024 Ronan Abhamon <ronan.abhamon@vates.tech> - 3.2.3-1.14
+* Fri Nov 29 2024 Ronan Abhamon <ronan.abhamon@vates.tech> - 3.2.3-1.13.0.pre.1
 - Sync fork-load-daemon script with http-nbd-transfer (v1.5.0)
-- WIP
+- Add many changes to support mypy
```

Comment:
Could be worded better, indicating the nature of the changes. "Numerous code improvements for issues detected by mypy"?